data_architect_misc/data_transformer/configs/LATAM_Brazil/step5_(Optional)_transform_raw_Search_investment_data.json

```diff
@@ -0,0 +1,121 @@
+[
```

Comment:
Please rename this file. Never include '(' or ')' in file names because they will cause trouble when run on Linux machines.